andela/ah-django-unchained-frontend
# #162948806 create draft article

#### What does this PR do?
Creates a user Interface through which a user can be able to create and save an article as a draft.

#### Description of Task to be completed?
* [x]   Create a folder structure
* [x]   Configure Redux and thunk middleware
* [x]   Consume the `post article` API endpoint

#### How should this be manually tested?
1. Clone this repository on your local machine using
   `git clone https://github.com/andela/ah-django-unchained-frontend.git`
2. Change your working directory to the project's directory using
   `cd ah-django-unchained-frontend`
3. Change to the specific branch `ft-create-draft-article-162948806 ` using
   `git checkout ft-create-draft-article-162948806 `
4. Create a `.env` file and inside it add this line `REACT_APP_BASE_URL=app-base-url` where `app-base-url ` is the root url of the backend API
5. Run `npm install` to install all the modules listed in the `package.json` file.
6. Run `npm start` command to start the application.
7. The application should start successfully. On the browser, navigate to the URL `http://localhost:3000/createArticle`, .

#### Link to PT story
[#162948806 ](https://www.pivotaltracker.com/story/show/162948806)
